### ISSUE TITLE
Update Mondo ontology - glygen/downloads/mondo/current (current file is a folder now but file remains the same)

### ISSUE THREAD

**Issue by jeet-vora** [glygener/glygen-frontend#156]
- glygen/downloads/mondo/current (current file is a folder now but file remains the same)

**Comment by rykahsay** [comment]
what is mondo ontology?

**Comment by rykahsay** [comment]
I don't know why you have assigned back to me without answering my question -- What exactly do you mean by "update mondo ontology"? What dataset file or any other file should be updated? What dataset was being created using file(s) under downloads/mondo/current/ ???

**Comment by jeet-vora** [comment]
This is the mondo file in downloads/mondo/current/**mondo.owl**
The processing workflow remains the same.

**Comment by rykahsay** [comment]
No, protein_disease_idmap.csv and protein_disease_names.csv depend on downloads/ohsu/mondo.json. I have never used downloads/mondo/current/mondo.owl

<img width="772" alt="image" src="https://user-images.githubusercontent.com/20584876/213034390-325e9330-d394-48ab-aaf4-b63b49a92e25.png">

**Comment by jeet-vora** [comment]
The file is in /data/projects/glygen/downloads/mondo/current/mondo.json
@rykahsay

**Comment by rykahsay** [comment]
done
